control-plane/pkg/apis/internals/kafka/eventing/v1alpha1/consumer_group_types.go

```diff
@@ -93,6 +93,13 @@ type ConsumerGroupStatus struct {
 	// DeliveryStatus contains a resolved URL to the dead letter sink address, and any other
 	// resolved delivery options.
 	eventingduckv1.DeliveryStatus `json:",inline"`
+
+	// Replicas is the desired number of replicas of the given Template.
+	// These are replicas in the sense that they are instantiations of the
+	// same Template, but individual replicas also have a consistent identity.
+	// If unspecified, defaults to 1.
+	// +optional
+	Replicas *int32 `json:"replicas,omitempty"`
```

Comment:
The `replicas` field in the status shouldn't be the "desired` number of replicas.

maybe the latest observed number of replicas or we can take a look at the replicas field in the status of the `Deployment` resource.